client/extensions/wp-job-manager/state/reducer.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Internal dependencies
+ */
+import { combineReducers } from 'state/utils';
+import settings from './settings/reducer';
+
+export default combineReducers( {
```

Comment:
Theoretically unneeded as long as there's only one reducer. There's a (slim?) chance we can rely on the `post-types` Redux subtree for the non-settings stuff, so might be worthwhile dropping `combineReducers` for now.